### ISSUE TITLE
Request : French translation

### ISSUE THREAD

**Issue by Lord-Schwartzy** [AgmasGold/anno-designer#62]
Would be nice to have the french traduction in the anno designer. I could help for it if you need too!

My discord : Schwartzy#3217

**Comment by AgmasGold** [comment]
Edited the title to "Request : French traduction" -> "Request : French translation"

**Comment by AgmasGold** [comment]
Waiting on PR's #67, #63

**Comment by AgmasGold** [comment]
The french translation is missing for 18 buildings within the presets file:

* gene_pharming_laboratory
* sponges_harvesting_station
* neurotechnology_factory
* energy_cell_factory
* lithium_harvesting_station
* bio_suite_factory
* equipment_factory
* coral_extract_factory
* black_smoker_miner_platinum
* coral_breeder
* energy_transmitter
* defence_platform
* gene_pharming_field
* sponges_harvester_field
* lithium_harvesting_field
* coral_field
* monument_techs
* underwater markethouse II

**Comment by StingMcRay** [comment]
On it to check the missing Translations for above buldings

**Comment by StingMcRay** [comment]
Fixed as in the preset file 3.2.2 that will come with the new version (in source already)
